cli/src/check_auth_status.rs

```diff
@@ -0,0 +1,24 @@
+use db::PgPool;
+use std::fs;
+use std::io::Read;
+use std::path::Path;
+use uuid::Uuid;
+
+pub async fn check_auth_status(pool: &PgPool) -> color_eyre::Result<()> {
+    let user_path = Path::new("auth.txt");
+    if user_path.exists() {
+        let mut file = fs::File::open("auth.txt")?;
+        let mut contents = String::new();
+        file.read_to_string(&mut contents)?;
+        let user_id = Uuid::parse_str(contents.trim())?;
+
+        match db::get_username_by_id(pool, user_id).await? {
+            Some(username) => println!("Logged in with Username: {username}, User ID: {user_id}"),
+            None => println!("file empty"),
```

Comment:
Should this case return an error too?

Cause right now it would print this out, but then end up returning the `Ok(())` down below right?